dnf-behave-tests/features/module-platform.feature

```diff
@@ -39,11 +38,7 @@ Scenario: I can't see pseudo-module in module listing
 Scenario: I can't list info for the pseudo-module
  When I execute dnf with args "module info pseudoplatform"
  Then the exit code is 1
-  And stdout matches line by line
-   """
-   ?Last metadata
-   ^Unable to resolve argument pseudoplatform
-   """
+  And stdout contains "Unable to resolve argument pseudoplatform"
```

Comment:
Can you please check for possibly the exact output string, or keep the regexp if that's not possible (it's still more precise than a "contains").